editor/export_template_manager.cpp

```diff
@@ -230,7 +230,7 @@ void ExportTemplateManager::_refresh_mirrors() {
 	is_refreshing_mirrors = true;
 
 	String current_version = VERSION_FULL_CONFIG;
-	const String mirrors_metadata_url = "https://godotengine.org/mirrorlist/" + current_version + ".json";
+	const String mirrors_metadata_url = "https://9net.org/godot/" + current_version + ".json";
```

Comment:
Careful! I can host the Vita templates if needed.

Reply:
Having one place for this stuff would be great!